src/style.css

```diff
@@ -1 +1,54 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  width: 1024px;
+  height: 60px;
+  justify-content: space-between;
+}
```

Comment:
remove width for the container, it should be expanded for the whole browser window
add horizontal padding instead